content/en/service_management/workflows/guide/private_action_credentials.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Handling Private Action Credentials
+kind: guide
+disable_toc: false
+---
+
+{{< callout url="https://google.com" btn_hidden="false" header="Join the Beta!">}}
+Private Actions are in beta. Use this form to request access today.
+{{< /callout >}}
+
+## Overview
+
+Some private actions such as Jenkins and PostgreSQL require credentials to function.
+- Create a JSON file for the credential and use the JSON structure provided in [Credential files](#credential-files).
+- Store your credential files in the configuration directory you created during setup.
+- When you specify the path to the credential in your runner connection, use the path to the credential file on the container.
+
+{{< img src="service_management/private-runner-creds.png" alt="The path to the credential file is '/etc/dd-action-runner/creds/creds.pgpass'" style="width:80%;" >}}
+
+## Credential files
```

Comment:
we should provide examples for credentials for the private actions HTTP action as well since it also requires credential files